language-server/modules/langserver-core/src/test/resources/codeaction/create-variable/source/createVariableWithCheck3.bal

```diff
@@ -0,0 +1,25 @@
+public type Error1 E1|E2;
+public type E1 distinct error;
+public type E2 distinct error;
+
+function foo1() {
+    getValueOrError1();
+}
+
+function getValueOrError1() returns int|Error1 => 1;
+
+public type E3 distinct error;
+public type Error2 E3|int;
+public type Error Error1|Error2;
+
```

Comment:
We need to add a negative test where on union member of the error is non-public and its a union with another non-error type.

```ballerina
type Error3 E3|int;
public type NonPublicError Error1|Error3;

function getValueOrError2() returns NonPublicError => 1;
```

Reply:
Added